commands/waitForAjaxCompleted.js

```diff
@@ -3,7 +3,7 @@ exports.command = function(callback) {
 
     // Uses waitForCondition to run code within the client browser
     //
-    return client.waitForCondition('return $.active;', 8000, function(result) {
+    return client.waitForCondition('return jQuery.active;', 8000, function(result) {
```

Comment:
Behaviour will be the same as the current behaviour when `$.active` is `undefined` - the value returned will be `null` instead of a number. In the logs this looks like:

```
INFO Request: POST /wd/hub/session/81930a77-f594-468f-a18d-ddb6661c1e99/execute 
 - data:  {"script":"return $.active;","args":[]} 
 - headers:  {"Content-Type":"application/json; charset=utf-8","Content-Length":39}
INFO Response 200 POST /wd/hub/session/81930a77-f594-468f-a18d-ddb6661c1e99/execute{ sessionId: '81930a77-f594-468f-a18d-ddb6661c1e99',
  status: 0,
  state: 'success',
  value: null,
  class: 'org.openqa.selenium.remote.Response',
  hCode: 1602572718 }
 ✔ Condition was satisfied after 17 milliseconds.
```

Compare to when `jQuery.active` is defined:
```
INFO Request: POST /wd/hub/session/2be1f58d-dc13-4e49-81ab-dff31a9b3d70/execute 
 - data:  {"script":"return jQuery.active;","args":[]} 
 - headers:  {"Content-Type":"application/json; charset=utf-8","Content-Length":44}
INFO Response 200 POST /wd/hub/session/2be1f58d-dc13-4e49-81ab-dff31a9b3d70/execute{ sessionId: '2be1f58d-dc13-4e49-81ab-dff31a9b3d70',
  status: 0,
  state: 'success',
  value: 0,
  class: 'org.openqa.selenium.remote.Response',
  hCode: 571248306 }
 ✔ Condition was satisfied after 16 milliseconds.
```